src/python/WMCore/Services/PyCondor/PyCondorAPI.py

```diff
@@ -44,6 +44,30 @@ def getCondorJobs(self, constraint, attr_list):
 
         return jobs
 
+    def editCondorJobs(self, job_spec, attr, value):
+        """
+        _editCondorJobs_
+
+        Edit a set of condor jobs given an attribute and value
+        job_spec can be a list of job IDs or a string specifying a constraint
+        """
+        success = False
+        try:
+            self.schedd.edit(job_spec, attr, value)
+            success = True
+        except Exception as ex:
+            # edit doesn't distinguish between an error and not matching any jobs
+            # check for this message and assume it just didn't match any jobs
+            if isinstance(ex, RuntimeError) and str(ex) == "Unable to edit jobs matching constraint":
+                success = True
+                msg = "Condor constraint did not match any jobs. "
```

Comment:
this should be a logging.info only. For the else problem, then I agree with logging.exception